examples/structured_2d_dgsem/elixir_eulerpolytropic_wave.jl

```diff
@@ -0,0 +1,88 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the polytropic Euler equations
+
+gamma = 1.001   # Quasi-isotropic simulation. gamma = 1.0 would lead to a singularity.
+# gamma = 2.0   # Adiabatic monatomic gas in 2d.
+kappa = 1.0     # Scaling factor for the pressure.
+equations = PolytropicEulerEquations2D(gamma, kappa)
+
+# Linear pressure wave in the negative x-direction.
+function initial_condition_wave(x, t, equations::PolytropicEulerEquations2D)
+  rho = 1.0
+  v1 = 0.0
+  if x[1] > 0.0
+    rho = ((1.0 + 0.01*sin(x[1]*2*pi)) / kappa)^(1/gamma)
+    v1 = ((0.01*sin((x[1]-1/2)*2*pi)) / kappa)
+  end
+  v2 = 0.0
+
+  return prim2cons(SVector(rho, v1, v2), equations)
+end
+initial_condition = initial_condition_wave
+
+volume_flux = flux_winters_etal
+solver = DGSEM(polydeg=2, surface_flux=flux_hll,
```

Comment:
```suggestion
solver = DGSEM(polydeg=3, surface_flux=flux_hll,
```
We typically have one elixir with at least `polydeg=3` for "true" high order. But this is just a suggestion...

Reply:
I agree. We should verify that a high-order approximation is achieved for the new equation system